Makefile

```diff
@@ -38,7 +38,7 @@ ifeq ("$(WITH_RACE)", "1")
 	BUILD_CGO_ENABLED := 1
 endif
 
-LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(shell git describe --tags --dirty --always)"
+LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=v6.5.0"
```

Comment:
ignore this line. I'll revert it

Reply:
without this change, when I started pd-server with local compiled build, it always complained the version number isn't a triple dot format. $(shell git describe --tags --dirty --always) gave me the hash. Is anyone know how I can start pd-server with local build without this change?

Reply:
Could you please show the actual error message?

Reply:
The problem has gone after I synced recent commits in tikv/pd. Now `git describe --tags --dirty --always` returned v6.6.0-alpha-74-g58a7b1f7, while it returned a hash code before the sync.

Reply:
I guess before we upgraded to v6.6.0-alpha and updated the tag, some tags had problem which caused the version info returned in the wrong format.